pexpect/async.py

```diff
@@ -30,10 +31,15 @@ def __init__(self, expecter):
     def found(self, result):
         if not self.fut.done():
             self.fut.set_result(result)
+            self.transport.pause_reading()
```

Comment:
I should be embarrassed... I am the one who first proposed/contributed the ``transport.pause_reading`` method to the asyncio library :)